main/src/ca/uwaterloo/flix/api/lsp/provider/CompleteProvider.scala

```diff
@@ -54,8 +116,34 @@ object CompleteProvider {
+    CompletionItem("let*", "let* ${1:name} = ${2:exp};\n", None, Some("code snippet for let*"), CompletionItemKind.Snippet, InsertTextFormat.PlainText, Nil),
+    CompletionItem("main", "def main(args: Array[String]) : Int32 & Impure = \n    ${1:/* code */} \n    0", None, Some("code snippet for main ()"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("main Hello World", "def main(_args: Array[String]) : Int32 & Impure = \n    println(\"Hello World!\");\n    0", None, Some("code snippet for Hello World Program"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("match", "match ${1:exp} {\n    case ${2:pat} => ${3:exp}\n}", None, Some("code snippet for pattern match"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("namespace", "namespace ${1:name} {\n    ${2:/* code */} \n}", None, Some("code snippet to create namespace"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("opaque", "opaque type ${1:name} = ${2:type}", None, Some("code snippet for opaque type"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("pub def", "pub def ${1:name}(${2:arg}:${3:type}): ${4:return_type} = \n", None, Some("code snippet to define a public function"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("query", "query ${1:db} select ${2:cols} from ${3:preds} ${4:where ${5:cond}}", None, Some("code snippet for query"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("rel", "rel ${1:name}(${2:x}: ${3:type}, ${4:y}: ${5:type})", None, Some("code snippet to declare predicate symbol"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("select", "select {\n    $1\n}", None, Some("code snippet for select"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("type alias", "type alias ${1:name} = ${2:type}", None, Some("code snippet for type alias"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("use", "use ${1:module};", None, Some("code snippet for use"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("law", "law ${1:name}: forall(${2:params}).${3:exp}", None, Some("code snippet for law"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("class", "pub class ${1:name}[${2:clause}] {\n    $3\n}", None, Some("code snippet for class"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("force", "force ${1:t}", None, Some("code snippet for force"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil),
+    CompletionItem("project", "project ${1:exp} into ${2:fixPoint}", None, Some("code snippet for project"), CompletionItemKind.Snippet, InsertTextFormat.Snippet, Nil)
+
   )
```

Comment:
I would like to replace "code snippet" -> "snippet" everywhere (just to keep things short).